scripts/build_proto.sh

```diff
@@ -17,11 +17,11 @@ find ${PROTO_IN} -name "*.proto" -print0 | xargs -0 protoc --python_out=${PYTHON
 if [ "$(uname)" == "Darwin" ]; then
     echo "Darwin"
     # note the '' for empty file on MacOS
-    find src/syft/proto -name "*_pb2.py" -print0 | xargs -0 sed -i '' 's/from \(proto.*\) import /from syft.\1 import /g'
```

Comment:
Any reason for looking for this files outside of the /proto folder?

Reply:
No, did not intend to push this change.